.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
```

We should use a link that includes closed issues:

```suggestion
            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc),
```